### PR TITLE
Improve CI jobs

### PR DESCRIPTION
**Pull request by marandaneto**
## :scroll: Description
_#skip-changelog_

## :pencil: Checklist

- [ ] I reviewed submitted code
- [ ] I added tests to verify changes
- [ ] No new PII added or SDK only sends newly added PII if `sendDefaultPii` is enabled
- [ ] I updated the docs if needed
- [ ] All tests passing
- [ ] No breaking changes

### CODE REVIEW

**Comment by marandaneto** [comment]
Formatting is fixed by https://github.com/getsentry/sentry-dart/pull/1463

File: .github/workflows/dart.yml [hunk 1]
```diff
@@ -34,35 +34,35 @@ jobs:
...
           - os: macos-latest
             sdk: beta
     steps:
       - uses: dart-lang/setup-dart@6a218f2413a3e78e9087f638a238f6b40893203d # pin@v1
         with:
           sdk: ${{ matrix.sdk }}
       - uses: actions/checkout@v3
-      # coverage with 'chrome' platform hangs the build
+
       - name: Test (VM and browser)
```
Comment (krystofwoldrich) [inline .github/workflows/dart.yml:47]: L: Was this not working with macOS before?

**Review by krystofwoldrich — APPROVED** [review]
I have one question above.

But besides that LG 🚀

File: .github/workflows/dart.yml (hunk 1 above)
Comment (marandaneto) [reply .github/workflows/dart.yml:47]: yes, that's what the comment says:

> coverage with 'chrome' platform hangs the build

For `macOS` specifically.

**Pull request merged**